parser.y

```diff
@@ -4136,7 +4140,9 @@ IndexOptionList:
 			} else if opt2.Tp != 0 {
 				opt1.Tp = opt2.Tp
 			} else if opt2.KeyBlockSize > 0 {
-			    opt1.KeyBlockSize = opt2.KeyBlockSize
+			    	opt1.KeyBlockSize = opt2.KeyBlockSize
+			} else if opt2.Invisible {
+				opt1.Invisible = opt2.Invisible
```

Comment:
🤔 

In MySQL the following should create a _visible_ index (the last option takes effect):

```sql
create table a (a int, key(a) invisible visible);
```